spring-cloud-gcp-pubsub/src/test/java/org/springframework/cloud/gcp/pubsub/core/subscriber/PubSubSubscriberTemplateTests.java

```diff
@@ -180,12 +175,12 @@ public void setUp() throws InterruptedException, TimeoutException, ExecutionExce
 				.addReceivedMessages(ReceivedMessage.newBuilder().setMessage(this.pubsubMessage).build()).build());
 
 		// create object under test
-		this.pubSubSubscriberTemplate = new PubSubSubscriberTemplate(this.subscriberFactory);
+		this.pubSubSubscriberTemplate = spy(new PubSubSubscriberTemplate(this.subscriberFactory));
 		this.pubSubSubscriberTemplate.setMessageConverter(this.messageConverter);
 	}
 
 	@Test
-	public void testSubscribeAndAck() throws InterruptedException, ExecutionException, TimeoutException {
+	public void testSubscribe_AndAck() throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
What drive you to do this renaming?
Why not `test_subscribe_and_ack()` then?

Reply:
This all came out from the original `testPullAndAck` which wasn't testing the `pullAndAck` method, but testing `pull` followed by `ack`. What's the convention that we should follow?

Reply:
Well, for consistency it should not be with the `_` - only cameCase.
But that's my preference so far...
Feel free to leave it as is if it really makes life easier...